src/logic/operate.js

```diff
@@ -0,0 +1,9 @@
+function toggleSign(result) {
+  if (result !== '') {
+    const num = parseFloat(result);
+    return (-num).toString();
+  }
+  return result;
+}
+
+export default toggleSign;
```

Comment:
Same here, Kindly make sure that you are using the operate.js file that is provided in the project requirements.

here is the [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)

Reply:
![image](https://github.com/AbdelazizSoliman/Math-Magicians/assets/51516029/9f1ed23e-d030-4373-ad8d-f63fd27c9fba)

I already imported both files, Kindly explain more to fix my issues to go to the next project, thank you.

Reply:
In the project requirements you are already given operate.js and calculate.js files, you need to use them by copying their contents to your files. so that your operate and calculate file are the SAME as the given ones.

Reply:
Thank you.